osbuild/buildroot.py

```diff
@@ -329,19 +330,21 @@ def run(self, argv, monitor, timeout=None, binds=None, readonly_binds=None, extr
         READ_ONLY = select.POLLIN | select.POLLPRI | select.POLLHUP | select.POLLERR
         poller = select.poll()
         poller.register(proc.stdout.fileno(), READ_ONLY)
+
+        stage_origin = "stages/" + stage_name
```

Comment:
Can we get this from somewhere more canonical? I know we always take them from stages right now but maybe sometimes we don't? Also officially these are paths so `os.path.join("stages", stage_name)`?